### DISCUSSION TITLE
Request Collections: can't sort, can't filter

### DISCUSSION THREAD

**Discussion by dasmith232** [Kong/insomnia#6278] [Help] [answered]
My Insomnia client just updated (automatically) to version 2023.5.0, running on Windows 10.0.22621. I've been using Insomnia for a few months now and have several dozen requests defined. Following today's update, the filter box at the top of the request list does not work. Nothing seems to happen when entering anything in that box. Also, the sort option has no effect. My requests are in an arbitrary order and navigating within the requests is essentially unusable.

I've restarted the program; no change. I've rebooted my machine; no change.

**Comment by DanielRouxSA** [comment]
I am encountering this as well. FWIW, I have filed two issues for these - #6284 and #6288.

**Comment by paulo156** [comment]
Having the same issue in Insomnia 2023.5.3.
Additionally, it looks like it's working really slow, if I open/close a folder of requests it takes a lot to load.
I have version 2023.4.0 on a different PC and over there I don't have these issues

**Comment by JohnBarhorst-mastery** [comment]
I just updated automatically to 2023.5.5, and I'm experiencing the same issues on an M2 MacBook. All my folders ordering was changed, and opening folders is painfully slow. Drag and drop ordering as well as the sort option (which I'd never even tried until now) do not make any changes.

On the plus side, I was inspired to start removing a lot of older requests and folders. It seems to have helped a little on the performance of opening folders. I don't even know what version I was on before, probably last updated a month ago or so.

**Comment by filfreire** [comment, marked as answer]
Hey folks, we're working on a fix for both of these issues https://github.com/Kong/insomnia/issues/6284 and https://github.com/Kong/insomnia/issues/6288.
